app/app/calculations/punching.py

```diff
@@ -0,0 +1,212 @@
+    elif support_id == 2:   # słup krawędziowy Y
+        u00 = min(cx + 2 * cy, cx + 3 * d)
+    elif support_id == 3:   # słup narożny
+        u00 = min(cx + cy, 3 * d)
+    elif support_id == 4:   # ściana - naroże
+        u00 = 100           # TODO(LZ): poprawic!
+        ERRORS.append("Podpora 'ściana-naroże' niedostępna.")
+    elif support_id == 5:   # ściana - koniec
+        u00 = 200           # TODO(LZ): poprawic!
+        ERRORS.append("Podpora 'ściana-koniec' niedostępna.")
+    else:
+        # TODO(LZ): dodac sciany (naroznik, koniec)
+        ERRORS.append('Błąd w określeniu położenia podpory (u0).')
+        u00 = 0.0
+    u0 = (1.0 - lambda_u / 100) * u00
+
+    if support_id == 0:     # slup wewnetrzny
+        u11 = 2 * cx + 2 * cy + 4 * math.pi * d
+    elif support_id == 1:   # słup krawędziowy X
+        u11 = 2 * cx + 1 * cy + 2 * math.pi * d + 2 * lx
+    elif support_id == 2:   # słup krawędziowy Y
+        u11 =  1 * cx + 2 * cy + 2 * math.pi * d + 2 * ly
+    elif support_id == 3:   # słup narożny
+        u11 = 1 * cx + 1 * cy + 1 * math.pi * d + lx + ly
+    elif support_id == 4:   # ściana - naroże
+        u11 = 100           # TODO(LZ): poprawic!
+        ERRORS.append("Podpora 'ściana-naroże' niedostępna.")
+    elif support_id == 5:   # ściana - koniec
+        u11 = 200           # TODO(LZ): poprawic!
+        ERRORS.append("Podpora 'ściana-koniec' niedostępna.")
+    else:
+        # TODO(LZ): dodac sciany (naroznik, koniec)
+        ERRORS.append('Błąd w określeniu położenia podpory (u1).')
+        u11 = 0.0
+    u1 = (1.0 - lambda_u / 100) * u11
+
+    # -----------------------------------------
+    # NAPREZENIA
+    # Maksymalne naprezenie przy przebiciu (vEd)
+    ved0 = beta * ved * (u0 * d)
+    ved1 = beta * ved * (u1 * d)
+
+    # Maksymalna wytrzymalosc na ścinanie przy przebiciu ze wzgledu na sciskane krzyzulce betonowe (vRdmax)
+    ni = 0.6 * (1.0 - fck / (250 * MPa))
+    vrdmax = 0.4 * ni * fcd
+
+    if vrdmax >= ved0:
+        ERRORS.append('Nośność betonu na ścinanie przy przebiciu zachowana (vEd0 <= vRdmax).')
+    else:   # elif vrdmax < ved0:
+        ERRORS.append('Nośność betonu na ścinanie przekroczona (vEd0 > vRdmax)!')
+
+    # Nośność na ścinanie przy przebiciu bez zbrojenia na przebicie (vRdc)
+    crdc = 0.18 / gamma_c
+    k = min(1.0 + (200 * mm / d)**0.5, 2.0)
+    ro_lx = asx / ((cy + 6 * d) * d)
+    ro_ly = asy / ((cx + 6 * d) * d)
+    ro_l = min((ro_lx * ro_ly)**0.5, 0.02)
+    vmin = 0.035 * k**(3/2) * (fck / MPa)**0.5 * MPa
+    k1 = 0.1
+    sigma_cp = 0.0 * MPa		# rozwinac !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+    vrdc = max(crdc * k * (100 * ro_l * fck / MPa)**(1/3) * MPa + k1 * sigma_cp, vmin + k1 * sigma_cp)
+
+    # -----------------------------------------
+    # ZBROJENIE
+
+    if vrdc >= ved1:
+        ERRORS.append('Nośność na ścinanie przy przebiciu zachowana (vEd1 <= vRdc).')
+        print('Nośność na ścinanie przy przebiciu zachowana (vEd1 <= vRdc).')
```

Comment:
Zbędny print? Chyba można to pousuwać. Jak wystąpi ten błąd, to rzuci go do templatki